src/zeep/wsdl/wsdl.py

```diff
@@ -106,43 +107,43 @@ def load(self, location):
     def __repr__(self):
         return "<WSDL(location=%r)>" % self.location
 
-    def dump(self):
-        print("")
-        print("Prefixes:")
+    def dump(self, output=sys.stdout):
+        print("", file=output)
```

Comment:
You can use local redefinition of print to avoid to set multiple `file=output`:

```python
from functools import partial

print = partial(print, file=output)

print("")
print("Prefixes:")
```

and `file` (as used in print) is better than `output`.

Reply:
@frost-nzcr4 I will performe the changes and submit another pull request.